renderer/configuration/robotouille_config.json

```diff
@@ -237,6 +308,36 @@
                 }
             }
         }
+    },
+    "condiment": {
+        "constants": {
+            "STATION_CONDIMENT_OFFSET" : 0.25,
+            "X_SCALE_FACTOR": 0.125,
+            "Y_SCALE_FACTOR": 0.75
+        },
+        "entities": {
+            "ketchupbottle": {
+                "assets": {
+                    "default": "ketchupbottle.png",
+                    "predicates": ["isketchup"]
```

Comment:
The default images shouldn't need any predicates, can refer to items like bread or topbun with only one image to see how it should be done